ariadne/cardano/src/Ariadne/Wallet/Cardano/Kernel/DB/AcidState.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE RankNTypes, TemplateHaskell #-}
+{-# OPTIONS_GHC -fno-warn-deriving-typeable #-}
```

Comment:
The reason for this is that `acid-state` adds a `deriving Typeable` clause in its `makeAcidic` TH wrapper.